lesson-17/index.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+
+// HW-17
+
+// task 1
+
+// const a = ("'true'");
+// const b = Boolean(NaN);
+// const c = (17);
+// const d = undefined;
+// const e = null;
+// console.log(typeof a);
+// console.log(typeof b);
+// console.log(typeof c);
+// console.log(typeof d);
+// console.log(typeof e);
+
+
+// task 2
+
+// const height = 15;
+// const width = 20;
+
+// if (height > width) {
+//     console.log(height);
+// } else if (width > height) {
+//     console.log(width);
+// }
+
+
+// task 3
+
+// for (let i = 1; i <= 20; i+=1) {
+//     if (i % 3 == 0) {
```

Comment:
Используй строгое сравнение, чтобы быть профессионалом
```suggestion
//     if (i % 3 === 0) {
```